src/api/sso/src/slack-bolt-client.js

```diff
@@ -0,0 +1,77 @@
+const { logger } = require('@senecacdot/satellite');
+const { App } = require('@slack/bolt');
+
+const { CHANNEL_ID } = process.env;
+const createSlackApp = () => {
+  const { SLACK_BOT_TOKEN, SIGNING_SECRET } = process.env;
+  if (!(SLACK_BOT_TOKEN && SIGNING_SECRET && CHANNEL_ID)) {
+    return false;
+  }
+  const app = new App({
+    token: SLACK_BOT_TOKEN.trim(),
+    signingSecret: SIGNING_SECRET.trim(),
+  });
+  return app;
+};
+const constructMessageBlocks = (displayName, githubUsername, blogUrl) => {
+  const messageBlocks = [
+    {
+      type: 'header',
+      text: {
+        type: 'plain_text',
+        emoji: true,
+        text: 'New Member Notification :tada:',
+      },
+    },
+    {
+      type: 'divider',
+    },
+    {
+      type: 'context',
+      elements: [
+        {
+          type: 'image',
+          image_url: `https://github.com/${githubUsername}.png?size=64`,
+          alt_text: 'avatar',
+        },
+        {
+          type: 'mrkdwn',
+          text: `*${displayName}* has joined the Telescope community. Be the first one to welcome them.`,
+        },
+        {
+          type: 'mrkdwn',
+          text: `<https://www.github.com/${githubUsername} | *Visit Github Profile*>`,
+        },
+        {
+          type: 'mrkdwn',
+          text: `<${blogUrl} | *Visit Blog*>`,
+        },
+      ],
+    },
+  ];
+  return messageBlocks;
+};
+const publishSignUpMessage = async (displayName, githubUsername, blogUrl) => {
+  if (!(displayName && githubUsername && blogUrl))
+    throw new Error(
+      'displayName, gitHubUsername, and blogUrl are required to send sign up announcement'
+    );
+  const app = createSlackApp();
+  if (!app) logger.info('Missing Slack Bolt env vars - skipping sign up announcement');
+  else {
+    try {
+      const messageBlocks = constructMessageBlocks(displayName, githubUsername, blogUrl);
+
+      const result = await app.client.chat.postMessage({
+        channel: CHANNEL_ID.trim(),
+        text: `${displayName} has joined the Telescope community. Be the first one to welcome them.`,
+        blocks: messageBlocks,
+        unfurl_links: false,
+      });
+      logger.debug(result);
```

Comment:
You need more context in your logs.  `logger.debug({ result }, 'Slack Bot message sent')` or something.

Reply:
Resolved in [7f17ce0](https://github.com/Seneca-CDOT/telescope/pull/3773/commits/7f17ce0cb32cb469696e4a53bff8ed52589a9a45).